Cargo.toml

```diff
@@ -32,3 +32,11 @@ pem = ["alloc", "der/pem"]
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
+
+[patch.crates-io]
+cms = { git = "https://github.com/RustCrypto/formats.git" }
+const-oid = { git = "https://github.com/RustCrypto/formats.git" }
+der = { git = "https://github.com/RustCrypto/formats.git" }
+der_derive = { git = "https://github.com/RustCrypto/formats.git" }
+spki = { git = "https://github.com/RustCrypto/formats.git" }
+x509-cert = { git = "https://github.com/RustCrypto/formats.git" }
```

Comment:
This one is on me. I need to complete the new API (which are going to bring an API break) and then I can publish a pre-release.